l3kernel/testfiles/m3sort002.lvt

```diff
@@ -29,6 +29,16 @@
     \NEWLINE
     \tl_sort:nN { {a\par b} } \ERROR
     \NEWLINE
+    \tl_sort:nN { 81 } \test_compare:nnTF
+    \NEWLINE
+    \tl_sort:nN { 815 } \test_compare:nnTF
+    \NEWLINE
+    \tl_sort:nN { {\if_true: 8\fi:}15 } \test_compare:nnTF
+    \NEWLINE
+    \tl_sort:nN { {\if_true: 8\else: 6\fi:}15 } \test_compare:nnTF
+    \NEWLINE
+    \tl_sort:nN { {\if_false: 8\else: 6\fi:}15 } \test_compare:nnTF
+    \NEWLINE
     \tl_sort:nN { 8{+2}3461{-0}2{00}3748 } \test_compare:nnTF
```

Comment:
It's a good idea to test primitive conditionals there.  Could you replace some of these by unbalanced conditionals (that are still valid integer expressions) such as `\use_none:n \if:w 7` or `0 \use_none:n \fi:`?